src/api/DBRequests.jsx

```diff
@@ -0,0 +1,15 @@
+import axios from 'axios';
+
+const API_BASE_URL = import.meta.env.VITE_API_BASE_URL;
+
+export const login = async ({ email, password }) => {
+  try {
+    const response = await axios.post(`${API_BASE_URL}/api/v1/login`, {
```

Comment:
I would use destructuring for `data` and `status` as we will only need those two items:
```javascript
const { data, status } = await axios.post(`${API_BASE_URL}/api/v1/login`, {
  email,
  password,
});
return { data, status };
```